basic/README.md

```diff
@@ -5,7 +5,7 @@ This project providers sample to show working with Spring Cloud Microservices.
 A Microserivce using Spring Cloud with a REST interface.
 
 * consumer
-A Microserivce using Spring Cloud with a REST interface. Consumer calls provider with RestTemplate.
+A Microserivce using Spring Cloud with a REST interface and Feign interface. Consumer calls provider with RestTemplate and Feign.
```

Comment:
A Microserivce using Spring Cloud with a REST interface. Consumer calls provider with RestTemplate and Feign.